tests/metrics/MetricsClients.integration.test.ts

```diff
@@ -1,9 +1,10 @@
+import type { MetricsClient } from '@dreamit/graphql-server-base'
+// eslint-disable-next-line @typescript-eslint/no-duplicate-imports
```

Comment:
Would it make sense to adjust that rule? Does it not support type imports?

Reply:
It supports type imports but has a conflict with the no-inline-type rule which states that within one import statement normal and type imports should not be mixed and a separate `import type` should be used, while no-duplicate-imports states that if there is an existing import for the source it should be reused. While the no-duplicate-imports is useful, e.g. when copy-pasting code, in these cases it could be ignored.